haddock-api/resources/html/Linuwial.std-theme/linuwial.css

```diff
@@ -408,7 +410,7 @@ blockquote {
 }
 
 .src {
-  background: #f4f4f4;
+  background: #f2f2f2;
```

Comment:
if really wonder if human can distinguish `#f2f2f2` from `#f4f4f4`, so this feels odd from development POV. But they can be different, I guess.

Reply:
Heh. I spent some time toggling CSS.